add ROOT cmake components RooStats and MathMore

I'm having trouble building analysis code that depends on Brufit, unless I require the ROOT components RooStats and MathMore (see linked PR). Without these components, when building my analysis code I get errors such as
```
brufit/lib/libbrufit.so: undefined reference to `RooStats::ModelConfig::GetWS() const'
brufit/lib/libbrufit.so: undefined reference to `RooStats::SPlot::~SPlot()'
brufit/lib/libbrufit.so: undefined reference to `vtable for RooStats::MetropolisHastings'
brufit/lib/libbrufit.so: undefined reference to `RooStats::MarkovChain::MarkovChain()'
...
```
I'm running:
- ROOT Version: 6.24/02
- cmake version 3.22.0
- g++ 7.5.0 (for building my analysis code)

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -11,7 +11,7 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 list(APPEND CMAKE_PREFIX_PATH $ENV{ROOTSYS}) ############USEROOTSYS
 
 #---Locate the ROOT package and defines a number of variables (e.g. ROOT_INCLUDE_DIRS)
-find_package(ROOT REQUIRED COMPONENTS Proof)  ###########USEROOTSYS
+find_package(ROOT REQUIRED COMPONENTS Proof RooStats MathMore)  ###########USEROOTSYS
 
 #---Define useful ROOT functions and macros (e.g. ROOT_GENERATE_DICTIONARY)
 include(${ROOT_USE_FILE}) ##########USEROOTSYS

